cypress/e2e/backoffice/file-manager-attachment/file-list.cy.ts

```diff
@@ -0,0 +1,136 @@
+      })
+      .then((fileContent) => {
+        testFiles.push({
+          fileContent,
+          fileName: 'image1.jpeg',
+          mimeType: 'image/jpeg',
+          filePath: 'cypress/fixtures/suite/backoffice/file-manager-attachment/test-files/image1.jpeg',
+        });
+      })
+      .then(() => {
+        cy.fixture('suite/backoffice/file-manager-attachment/test-files/image2.png', 'binary');
+      })
+      .then((fileContent) => {
+        testFiles.push({
+          fileContent,
+          fileName: 'image2.png',
+          mimeType: 'image/png',
+          filePath: 'cypress/fixtures/suite/backoffice/file-manager-attachment/test-files/image2.png',
+        });
+      })
+      .then(() => {
+        cy.fixture('suite/backoffice/file-manager-attachment/test-files/document2.pdf', 'binary');
+      })
+      .then((fileContent) => {
+        testFiles.push({
+          fileContent,
+          fileName: 'document2.pdf',
+          mimeType: 'application/pdf',
+          filePath: 'cypress/fixtures/suite/backoffice/file-manager-attachment/test-files/document2.pdf',
+        });
+      })
+      .then(() => {
+        fileManagerAttachmentAddPage.visit();
+        fileManagerAttachmentAddPage.uploadFiles(testFiles);
+        fileManagerAttachmentAddPage.verifyFileUploadConstraints();
+        fileManagerAttachmentAddPage.submitForm();
+        fileManagerAttachmentAddPage.verifySuccessMessage();
+      });
+  });
+
+  it('should successfully delete a file', () => {
+    const fileManagerAttachmentDeletePage = container.get(FileManagerAttachmentDeletePage);
+
+    fileManagerAttachmentListPage.visit();
+    fileManagerAttachmentListPage.clickDeleteButton();
+
+    fileManagerAttachmentDeletePage.confirmDelete();
+    fileManagerAttachmentDeletePage.verifySuccessMessage();
+  });
+
+  it('should display file details on view page', () => {
+    const fileManagerAttachmentViewPage = container.get(FileManagerAttachmentViewPage);
+
+    fileManagerAttachmentListPage.visit();
+    fileManagerAttachmentListPage.clickViewButton();
+    fileManagerAttachmentViewPage.verifyFileDetailsAreVisible();
+  });
+
+  it('should successfully attach file to multiple entities', () => {
+    const fileManagerAttachmentAttachPage = container.get(FileManagerAttachmentAttachPage);
+
+    fileManagerAttachmentListPage.visit();
+    fileManagerAttachmentListPage.clickAttachButton();
+    fileManagerAttachmentAttachPage.selectCompany('comp');
+    fileManagerAttachmentAttachPage.selectCompanyUser('spen');
+    fileManagerAttachmentAttachPage.selectCompanyBusinessUnit('unit');
+    fileManagerAttachmentAttachPage.submitForm();
+    fileManagerAttachmentAttachPage.verifySuccessMessage();
+  });
+});
+
+function describeForSsp(title: string, options: { tags: string[] }, fn: () => void): void {
+  (['suite'].includes(Cypress.env('repositoryId')) ? describe : describe.skip)(title, fn);
```

Comment:
please be sure that this tests will not be run for the core (for now it looks like this PR will affect core test CI scope)

Reply:
@kraal-spryker I'm doing the same in my PRs. We can align regarding this